lib/matplotlib/backend_tools.py

```diff
@@ -989,12 +989,10 @@ def trigger(self, *args, **kwargs):
                  'help': ToolHelpBase,
                  'copy': ToolCopyToClipboardBase,
                  }
-"""Default tools"""
```

Comment:
These are docstrings; they should probably be comments instead.

Reply:
I thought about converting them but they seemed redundant with eachother and with the variable name. Happy to add back though

Reply:
+1 for removing the docstrings are indeed redundant.